app/models/concerns/live_lookup_ids.rb

```diff
@@ -0,0 +1,24 @@
+# Mixin that adds an abstraction layer for bib/instance and item ids needed for live lookup
+# of availability to account for differences between Sirsi and FOLIO
+
+module LiveLookupIds
+  # For Sirsi use the catalog key for availability lookup; for FOLIO use the Instance UUID
+  def live_lookup_id
+    return self[:uuid_ssi] if Settings.FOLIO_LIVE_LOOKUP && fetch(:uuid_ssi, nil)
+
+    self[:id]
+  end
```

Comment:
Is there a case where we want to use FOLIO data, but hit Sirsi for availability? I'd be comfortable with
```suggestion
  def live_lookup_id
   self[:uuid_ssi] || self[:id]
  end
```

unless there's a good reason not to?